src/cursor/cursor.ts

```diff
@@ -828,10 +907,7 @@ export class Cursor<
         this.kill();
       }
 
-      this._endSession(() => {
-        this.emit(Cursor.CLOSE);
-        cb(undefined, this);
-      });
+      this._endSession(() => cb(undefined, this));
```

Comment:
I thought the idea was that the cursor would still emit `close` just for symmetry with `isClosed`? It also seems to me that a `CursorStream` created from this `Cursor` would need to listen to that event in order to proactively close itself as well.